src/Data/Variant/Internal.purs

```diff
@@ -32,24 +33,28 @@ import Control.Alternative (class Alternative, empty)
 import Data.List as L
 import Data.Maybe (Maybe(..))
 import Data.Maybe as M
-import Data.Symbol (class IsSymbol, SProxy(SProxy), reflectSymbol)
+import Data.Symbol (class IsSymbol, reflectSymbol)
 import Partial.Unsafe (unsafeCrashWith)
 import Prim.Row as R
 import Prim.RowList as RL
 import Record.Unsafe (unsafeGet, unsafeHas) as Exports
 import Type.Data.Row (RProxy(..))
 import Type.Data.RowList (RLProxy(..))
+import Type.Proxy (Proxy(..))
 import Type.Equality (class TypeEquals)
 
 -- | Proxy for a `Functor`.
+-- |
+-- | **Deprecated** Use `Type.Data.Proxy (Proxy)` instead.
```

Comment:
Is this right? It's more that `FProxy` isn't needed at all anymore, right? If so, this is already a breaking release, so is there a sense in removing `FProxy` altogether? @natefaubion

Reply:
Sure, I guess that makes sense.

Reply:
I don't have a strong preference; if you'd rather keep it in then that's fine too.

Reply:
I didn't remove it because `FProxy` could be used by someone elsewhere (as opposed to them defining it in their own code). However, I think that's very unlikely and I'd rather remove it.